tests/conftest.py

```diff
@@ -34,6 +33,7 @@
 from kart.sqlalchemy.postgis import Db_Postgis  # noqa: E402
 from kart.sqlalchemy.sqlserver import Db_SqlServer  # noqa: E402
 from kart.sqlalchemy.mysql import Db_MySql  # noqa: E402
+from kart import subprocess_util as subprocess
```

Comment:
not sure this aliasing is a _great_ idea wrt functions that are almost-but-not the same as the stdlib ones.

Thoughts @craigds ?

Reply:
Happy to rename our `subprocess.run` to `kart_subprocess.run` or whatever but I'll do it in a follow up PR - will be even more clutter in an already cluttered PR

Reply:
I don't feel strongly about it TBH if you & Craig are good with it.

Reply:
kubprocess